migrator/migrator.go

```diff
@@ -382,29 +380,26 @@ func (m Migrator) MigrateColumn(value interface{}, field *schema.Field, columnTy
 			alterColumn = true
 		} else {
 			// has size in data type and not equal
-
 			// Since the following code is frequently called in the for loop, reg optimization is needed here
 			matches := regRealDataType.FindAllStringSubmatch(realDataType, -1)
 			matches2 := regFullDataType.FindAllStringSubmatch(fullDataType, -1)
-			if (len(matches) == 1 && matches[0][1] != fmt.Sprint(field.Size) || !field.PrimaryKey) && (len(matches2) == 1 && matches2[0][1] != fmt.Sprint(length)) {
+			if (len(matches) == 1 && matches[0][1] != strconv.Itoa(field.Size) || !field.PrimaryKey) &&
+				(len(matches2) == 1 && matches2[0][1] != strconv.FormatInt(length, 10)) {
```

Comment:
-1

Reply:
There is no need to make so many if nesting.